ctapipe/image/waveform_cleaning.py

```diff
@@ -67,6 +67,26 @@ def apply(self, waveforms):
         return waveforms
 
 
+class BaselineWaveformCleaner(WaveformCleaner):
+    """
+    Basic waveform cleaner that subtracts the waveform baseline as
+    estimated  in a window of "baseline_width" samples, shifted of
+    "window_shift" samples
+    """
+    baseline_width = Int(10, help='Define then number of samples for estimating the '
+                                  'baseline').tag(config=True)
+
+    window_shift = Int(0, help='Define the shift of the window on which calculate '
+                               'the baseline.').tag(config=True)
+
+    def apply(self, waveforms):
+        # Subtract initial baseline
+        baseline_sub = waveforms - np.mean(waveforms[:, :, self.window_shift:self.baseline_width],
```

Comment:
I think this should be `self.window_shift:self.window_shift + self.baseline_width` to get a window of length `self.baseline_width`

Reply:
Yes, surely, sorry. I added a test function. Should I change the arguments to sample_start sample_end?

Reply:
I have no opnion on what would be better, both solutions would be fine with me.